src/components/pages/import-batch-view.js

```diff
@@ -0,0 +1,171 @@
+      : <span>{t('taskStatusPending')}</span>;
+  };
+
+  const renderBatchStatus = ({ progress }) => {
+    if (progress === 1) {
+      return <span>{t('batchStatusDone')}</span>;
+    }
+    if (progress === 0) {
+      return <span>{t('batchStatusPending')}</span>;
+    }
+    return <span>{t('batchStatusProcessing', { progress: batch.progress })}</span>;
+  };
+
+  const renderDate = date => <span>{formatDate(date)}</span>;
+  const renderTitle = ({ title, slug }) => (
+    <a
+      target="_blank"
+      href={getImportedArticleUrl({ hostName, allowUnsecure, slug })}
+      rel="noreferrer noopener"
+      >{title}
+    </a>
+  );
+
+  const renderImportType = taskParams => {
+    let icon = null;
+    if (taskParams.importType === DOCUMENT_IMPORT_TYPE.add) {
+      icon = <CloudDownloadOutlined />;
+    }
+    if (taskParams.importType === DOCUMENT_IMPORT_TYPE.update) {
+      icon = <CloudSyncOutlined />;
+    }
+    return <Tooltip title={t(taskParams.importType)} className="ImportBatchViewPage-importType">{icon}</Tooltip>;
+  };
+
+  const renderErrorCount = ({ errors }) => <span>{errors?.length || 0} {t('errors')}</span>;
+
+  const renderErrors = ({ errors }) => (
+    <List
+      dataSource={errors}
+      renderItem={error => (
+        <List.Item>
+          <span>{error}</span>
+        </List.Item>)}
+      />);
+
+  const attemtpsTableColumns = [
+    { title: t('startedOn'), dataIndex: 'startedOn', width: '50px', render: renderDate },
+    { title: t('completedOn'), key: '_id', dataIndex: 'completedOn', width: '150px', render: renderDate },
+    { title: t('errorsCount'), width: '150px', render: renderErrorCount }
+  ];
+
+  const attemptsRenderer = task => (
+    <Table
+      key={task._id}
+      rowKey={attempt => `${task._id}_${attempt.startedOn}}`}
+      dataSource={task.attempts}
+      columns={attemtpsTableColumns}
+      pagination={false}
+      expandable={{
+        rowExpandable: attempt => attempt.errors?.length,
+        expandedRowRender: attempt => renderErrors(attempt)
+      }}
+      />
+  );
+
+  const renderHasErrrors = attempts => doesTaskHaveErrors(attempts) ? t('common:yes') : t('common:no');
+
+  const renderIsSuccessful = task => {
+    if (!task.processed) {
+      return <span />;
+    }
+
+    return isTaskSuccessful(task) ? t('common:yes') : t('common:no');
+  };
+
+  const taskTableColumns = [
+    { title: t('importType'), dataIndex: 'taskParams', width: '50px', render: renderImportType },
+    { title: t('documentTitle'), key: '_id', dataIndex: 'taskParams', width: '150px', render: renderTitle },
+    { title: t('taskStatus'), dataIndex: 'processed', width: '150px', render: renderStatus, sorter: task => task.processed ? 1 : -1 },
+    { title: t('hasErrors'), width: '150px', render: renderHasErrrors, sorter: task => doesTaskHaveErrors(task) ? 1 : -1 },
+    { title: t('isSuccessful'), width: '150px', render: renderIsSuccessful, sorter: task => isTaskSuccessful(task) ? 1 : -1 }
```

Comment:
These sorters aren't real sorters, they just do a grouping. One caveat of this particular one is that it doesn't distinguish between unsuccessful tasks and tasks that were not processed yet. If we want that I'd rather make another ticket for "proper sorters" and extend the functionality since this ticket has already a lot of new code.

Reply:
Sounds good. We could make a ticket to make all tables (that are not yet sortable) in the admin area sortable.

Reply:
It might be confusing for the user (at least I was) to have two columns "hasErrors" and "isSuccessful". Would any one of those ever be anything else than the opposite of the other? Maybe we can merge this into one column (yay/nay)?

Reply:
I've done the sorting better cause I was getting bored. Please look at the task-utils.

It's useful to see if you had attempts that had errors (I think for us it will be easier to follow that). The logic is that a task can have errors on one attempt but succeed on a subsequent one. The first column tells you if there were any errors at all, the last one is the final state (success or failure). We could drop the one that says "have we had errors at all" if you think it's too much.

Reply:
Can we please do this separately? 🙏 This PR has so many corrections that it reached 38 commits and 85 comments, I think the focus should be on stabilising it and getting it merged without risking side-effects.

Reply:
I agree, just saying that the column names are confusing, I think we shouldn't have two boolean columns with labels that are opposite to each other. But we could do one of the following:
* Rename "hasErrors" to "errors" and actually show the errors in that column
* Merge the two columns into one that can have one of the three values "succeeded" / "succeeded with errors" / "failed"
I would like to see the actual errors, so the first option would be my favorite